extensions/amp-story/1.0/test/test-amp-story-system-layer.js

```diff
@@ -141,4 +145,90 @@ describes.fakeWin('amp-story system layer', {amp: true}, (env) => {
     // Default "canonical"
     expect(shareButton.href).to.equal('http://localhost:9876/context.html');
   });
+
+  it('should show paused button if story has element with playback', () => {
+    systemLayer.build();
+    storeService.dispatch(Action.TOGGLE_STORY_HAS_PLAYBACK_UI, true);
+    expect(systemLayer.getShadowRoot()).to.have.class(
+      'i-amphtml-story-has-playback-ui'
+    );
+  });
+
+  it('should enable paused button if page has element with playback', () => {
+    systemLayer.build();
+    storeService.dispatch(Action.TOGGLE_STORY_HAS_PLAYBACK_UI, true);
+    storeService.dispatch(Action.TOGGLE_PAGE_HAS_ELEMENT_WITH_PLAYBACK, true);
+    expect(
+      systemLayer
+        .getShadowRoot()
+        .querySelector('button.i-amphtml-story-pause-control')
+    ).to.not.have.attribute('disabled');
+    expect(
+      systemLayer
+        .getShadowRoot()
+        .querySelector('button.i-amphtml-story-play-control')
+    ).to.not.have.attribute('disabled');
+  });
+
+  it('should disable paused button if page does not has elements with playback', () => {
+    systemLayer.build();
+    storeService.dispatch(Action.TOGGLE_STORY_HAS_PLAYBACK_UI, true);
+    storeService.dispatch(Action.TOGGLE_PAGE_HAS_ELEMENT_WITH_PLAYBACK, false);
+    expect(
+      systemLayer
+        .getShadowRoot()
+        .querySelector('button.i-amphtml-story-pause-control')
+    ).to.have.attribute('disabled');
+    expect(
+      systemLayer
+        .getShadowRoot()
+        .querySelector('button.i-amphtml-story-play-control')
+    ).to.have.attribute('disabled');
+  });
+
+  it('setting paused state to true should show the paused button', () => {
```

Comment:
shouldn't it show the "play" button when it's paused?

Reply:
True, the description was not matching the test even if the result is the same